mirai-core/src/commonMain/kotlin/network/protocol/packet/login/StatSvc.kt

```diff
@@ -84,6 +84,17 @@ internal class StatSvc {
         }
     }
 
+    internal object QueryHB : IncomingPacketFactory<Packet?>("StatSvc.QueryHB") {
+        override suspend fun ByteReadPacket.decode(bot: QQAndroidBot, sequenceId: Int): Packet? {
+            return null
+        }
+
+        override suspend fun QQAndroidBot.handle(packet: Packet?, sequenceId: Int): OutgoingPacket {
+            return Register.invoke(client, regPushReason = RegPushReason.serverPush)
+        }
```

Comment:
Kotlin 中, 目标方法为 `operator fun invoke(.....)` 时, 可以直接 `Register(client......)`

```suggestion
        override suspend fun QQAndroidBot.handle(packet: Packet?, sequenceId: Int): OutgoingPacket {
            return Register(client, regPushReason = RegPushReason.serverPush)
        }
```